CMakeLists.txt

```
@@ -70,6 +70,7 @@ math(EXPR DD4HEP_HIGH_MEM_POOL_DEPTH "${HOST_RAM_MB} / 2000" OUTPUT_FORMAT DECIM
 
 option(DD4HEP_USE_XERCESC       "Enable 'Detector Builders' based on XercesC"   OFF)
 option(DD4HEP_USE_GEANT4        "Enable the simulation part based on Geant4"    OFF)
+option(DD4HEP_USE_CELERITAS     "Enable offloading tracks to Celeritas" OFF)
```

```suggestion
option(DD4HEP_USE_CELERITAS     "Enable offloading tracks to Celeritas" OFF)
```
Please align the OFF